src/UserService.Mappers/RequestsMappers/UserRequestMapper.cs

```diff
@@ -12,7 +13,7 @@ public DbUser Map(UserRequest value)
         {
             if (value == null)
             {
-                throw new BadRequestException();
+                throw new BadRequestException(); 
             }
 
             value.Id ??= Guid.NewGuid();
```

Comment:
What does "??" mean?

Reply:
Sorry, that isn't my part of the code. My changes are marked "+"

Reply:
Oh, I see